tests/Js/ChangeEventTest.php

```diff
@@ -60,8 +60,14 @@ public function testSetValueChangeEvent($elementId, $valueForEmpty, $valueForFil
 
     public function setValueChangeEventDataProvider()
     {
-        $file1 = __DIR__ . '/../../web-fixtures/file1.txt';
-        $file2 = __DIR__ . '/../../web-fixtures/file2.txt';
+        // paths must be canonical and reachable from browser
```

Comment:
newer browsers, for security reasons, validate the paths if they exist and are canonical, of course from browser POV, thus we must use some paths available within the browser container

Reply:
This doesn't introduce any BC beaks for the Selenium2?

Reply:
#67 should fix this part.

Reply:
The combination of #67 and #88 will fix it.